csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegative/NumberValidation.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ValidateNumberIsPositiveOrNegative
+{
+    public static class NumberValidation
+    {
+        public static int IsPositiveOrNegativeUsingConditionalMethod(int number)
+        {
+            var result = 0;
+
+            if (number > 0)
+            {
+                result = 1;
+            }
+            else if (number < 0)
+            {
+                result = - 1;
+            }
+            else
+            {
+                result = 0;
+            }
+            return result;
+        }
+        public static int IsPositiveOrNegativeUsingLeftShiftMethod(int number)
+        {
```

Comment:
This method will return 1 for 0, it is inconsistent with the above method where it will return 0. Check if the number is zero then return 0, else do the left shift

Reply:
Updated method to add case for zero.